java-checks/src/test/files/checks/security/FileHandlingCheck.java

```diff
@@ -28,7 +28,7 @@ void fun(String strPath, String StrParent, String StrChild, String prefix, Strin
     new File(parent, StrChild); // compliant we rely on the fact that parent should be highlighted
 
 
-      File.createTempFile(prefix, suffix);
+    File.createTempFile(prefix, suffix); // Noncompliant
```

Comment:
I would add the other createTempFile method to enforce the fact that we explicitly target this one only.